src/korean.js

```diff
@@ -59,6 +59,9 @@ class Korean {
     if (word[wordLength - 2] === '하') {
       conjugate = word.slice(0, wordLength - 2);
       return conjugate.concat('해');
+    } else if (word[wordLength - 2] === '앉' || word[wordLength - 2] === '솟' || word[wordLength - 2] === '닫') {
```

Comment:
I agree with @songz. I also made a comment on the #47. If the final jamo comes after ㅏ and ㅗ, it will have 아, but if it comes after ㅓ and ㅜ, it will have 어. So, I also think it needs to be done with breakword.